mycore-neo4j/src/main/resources/xsl/mycoreobject-neo4j.xsl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
-  <xsl:output method="html" encoding="UTF-8" indent="yes"/>
+<xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
```

Comment:
This file resides in the `xsl` folder and should therefore be processable by Xalan. Afaik, that precludes an version other then `1.0`, or am I confused about that?